portfolio/src/main/webapp/style.css

```diff
@@ -106,6 +114,31 @@ td p {
   color: rgb(61,95,121);
 }
 
+#hobbies-div {
+  color: white;
+}
+
+.hobbies-page-a {
+  color: rgb(250,220,220);
+  text-decoration: none;
+}
+
+.hobbies-description {
+  padding: 3rem 4rem 3rem 1rem;
+}
+
+.hobbies-imgs tr td img {
+  height: 8rem;
+}
+
+.hobbies-image {
+  margin: .5rem;
+}
+
+.hobbies-description td {
```

Comment:
Not following alphabetic order here?

Reply:
Could you clarify what should be alphabetized? I was going to do another PR that alphabetized the attributes within each block, but should the blocks themselves be following some order?

Reply:
Yes, I guess both. @hlz03 Please correct me if I am wrong.

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order